README.md

```diff
@@ -1,133 +1,616 @@
-## Smithy Typescript
+# <img alt="Smithy" src="https://github.com/smithy-lang/smithy/blob/main/docs/_static/favicon.png?raw=true" width="28"> Smithy TypeScript
 
-Smithy code generators for TypeScript.
+> **WARNING: Smithy TypeScript is currently in [Developer Preview](https://aws.amazon.com/blogs/devops/smithy-server-and-client-generator-for-typescript). All interfaces and supported JavaScript platforms are subject to change.**
+> 
+> **NOTE: For Node.js only, currently versions >= 14 are supported, and are subject to change.**
 
-**WARNING: All interfaces are subject to change.**
+`smithy-typescript` includes the reference implementations of the [Smithy](https://smithy.io/) code generators for [TypeScript](https://www.typescriptlang.org/).
 
-## Generating a client
-
-This repository builds TypeScript declarations and JavaScript clients from Smithy
-models.
+For Client SDK code generation, the `typescript-client-codegen` plugin provides a framework for generating customizable TypeScript clients that can support multiple JavaScript platforms, including Node.js, Browser, and React-Native. See [the section on generating a client to see how to get started](#generating-a-client), or [the `typescript-client-codegen` documentation](#client-sdk-code-generation-typescript-client-codegen-plugin).
```

Comment:
This seems to conflict with the `For Node.js only,` note above

Reply:
Update the note to be more clear.